modules/extensions/src/path-style/shaders.glsl.js

```diff
@@ -49,7 +50,9 @@ float round(float x) {
       gapLength > 0.0 &&
       mod(clamp(vPathPosition.y, 0.0, vPathLength) + offset, unitLength) > solidLength
     ) {
-      discard;
+      if (!(allowPickGap && picking_uActive)) {
```

Comment:
Shouldn't discard if `allowPickGap` is `true`?

Reply:
Maybe I miss something, but what I'd like to achieve is allow picking on gaps, so I updated the discard logic, such that when `dashGapPickable` with picking active, the color of gaps won't be discarded.